docs/source/download.rst

```diff
@@ -9,45 +9,67 @@ Thetis requires installation of `Firedrake
 principle other Linux and Linux-like systems) and must be run from
 within the Firedrake virtual environment.
 
-Install Firedrake and Thetis
------------------------------
+Installing Firedrake
+---------------------
 
-You can install both Firedrake and Thetis by running::
+You can install Firedrake by following the download documentation on the
+`Firedrake website <http://firedrakeproject.org/download.html>`_.
 
-    curl -O https://raw.githubusercontent.com/firedrakeproject/firedrake/master/scripts/firedrake-install
-    python3 firedrake-install --install thetis
+.. note::
 
-See  `Firedrake website <http://firedrakeproject.org/download.html>`_ for more
-information on the installation procedure. Note that the install proceduce may
-take up to one hour depending on your system.
+   **Installing Firedrake can take up to an hour depending on the system.**
```

Comment:
But this isn't true! It's much much faster now. Maybe 20 minutes on a slow non-Ubuntu computer.

Reply:
Just you wait for https://github.com/firedrakeproject/firedrake/pull/4122! We are building PETSc in under 2 minutes